src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs

```diff
@@ -16,6 +16,7 @@
 using Dynamo.Logging;
 using Dynamo.UI;
 using Dynamo.ViewModels;
+using static Dynamo.UI.Controls.DynamoToolTip;
```

Comment:
No changes to this file, can remove the static usings.

Reply:
Done!